tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt

```diff
@@ -17,50 +27,123 @@ list(REMOVE_ITEM MICRO_SOURCES ${MICRO_TESTS_SOURCES})
 list(REMOVE_ITEM MICRO_KERNELS_SOURCES ${MICRO_TESTS_KERNELS_SOURCES})
 list(REMOVE_ITEM MICRO_MEMORY_SOURCES ${MICRO_TESTS_MEMORY_SOURCES})
 
-add_subdirectory(src)
-add_subdirectory(cmake_sub/simple_features)
-add_subdirectory(cmake_sub/micro_features)
-add_subdirectory(cmake_sub/micro_speech)
-add_subdirectory(cmake_sub/kiss_fft)
-add_subdirectory(cmake_sub/microfrontend)
+target_compile_options(app PRIVATE "-fno-threadsafe-statics")
 
-target_sources (
-	app PRIVATE
-        ${MICRO_SOURCES}
-        ${MICRO_KERNELS_SOURCES}
-        ${MICRO_MEMORY_SOURCES}
-        ${MICRO_HEADERS}
-        ${MICRO_KERNEL_HEADERS}
-        ${MICRO_MEMORY_HEADERS}
-        ${TF_MICRO_DIR}/../c/common.c
-        ${TF_MICRO_DIR}/../core/api/error_reporter.cc
-        ${TF_MICRO_DIR}/../core/api/flatbuffer_conversions.cc
-        ${TF_MICRO_DIR}/../core/api/op_resolver.cc
-        ${TF_MICRO_DIR}/../core/api/tensor_utils.cc
-      	${TF_MICRO_DIR}/../kernels/internal/quantization_util.cc
-        ${TF_MICRO_DIR}/../kernels/kernel_util.cc
-	)
-
-target_link_libraries(
-	app PRIVATE
-	zephyr_src
-	micro_features
-	simple_features
-	micro_speech
-	kiss_fft
-	microfrontend
-	)
+target_sources(
+  app
+  PRIVATE ${MICRO_SOURCES}
+          ${MICRO_KERNELS_SOURCES}
+          ${MICRO_MEMORY_SOURCES}
+          ${MICRO_HEADERS}
+          ${MICRO_KERNEL_HEADERS}
+          ${MICRO_MEMORY_HEADERS}
+          ${TF_LITE_DIR}/c/common.c
+          ${TF_LITE_DIR}/core/api/error_reporter.cc
+          ${TF_LITE_DIR}/core/api/flatbuffer_conversions.cc
+          ${TF_LITE_DIR}/core/api/op_resolver.cc
+          ${TF_LITE_DIR}/core/api/tensor_utils.cc
+          ${TF_LITE_DIR}/kernels/internal/quantization_util.cc
+		  ${TF_LITE_DIR}/kernels/kernel_util.cc
+		  
+          # zephyr-src
+          src/audio_provider.cc
+          src/audio_provider.h
+          src/command_responder.cc
+		  src/command_responder.h
```

Comment:
Can we have a consistent indentation?

Reply:
Sure we have, but I am already tired of fight with vs code and cli editors to have it done :). Believe me or not but I've downloaded special plugin for proper formatting and it was looking good locally. Git seems to break it. If you have time pls try to do the formatting. Maybe I could make use of some git configuration tips :p. (I think the point is in \r\n - \n convert.)

Reply:
Check my update in the latest commit on this